src/coreclr/src/inc/eventtracebase.h

```diff
@@ -204,8 +204,7 @@ struct ProfilingScanContext;
 
 #define ETWOnStartup(StartEventName, EndEventName) \
     ETWTraceStartup trace##StartEventName##(Microsoft_Windows_DotNETRuntimePrivateHandle, &StartEventName, &StartupId, &EndEventName, &StartupId);
-#define ETWFireEvent(EventName) \
-    ETWTraceStartup::StartupTraceEvent(Microsoft_Windows_DotNETRuntimePrivateHandle, &EventName, &StartupId);
+#define ETWFireEvent(EventName) FireEtw##EventName(GetClrInstanceId())
```

Comment:
@sywhang I believe you made a similar change for non-Windows recently.  Does this look right?  The event wasn't firing across EventPipe on Windows until I made this change.

Reply:
👍

Reply:
Scratch the thumbsup earlier... Can you check that after this change, getting the event via ETW works?

Reply:
Good idea! I'll do that .

Reply:
@josalem Were you able to verify that? I can do it if you're too busy with other parts of the PR :)

Reply:
If you've got some cycles, that would be much appreciated 😄.  I'm investigating these last couple test failures on Windows now.

Reply:
To document this: I wasn't able to collect this event via ETW with _and without_ my changes.  I think I'll fix this after the fact.